mods/creative/init.lua

```diff
@@ -163,7 +163,7 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	local formspec = player:get_inventory_formspec()
 	local filter = formspec:match("filter;;([%w_:]+)") or ""
 	local start_i = formspec:match("list%[detached:creative_".. player_name ..";.*;(%d+)%]")
-	local tab_id = tonumber(formspec:match("tabheader%[.*;(%d+)%;.*%]"))
+	local tab_id = tonumber(formspec:match("tabheader%[.*;(%d)%;.*%]"))
```

Comment:
Yes, but if the number of tabs in increased later, it is best to keep the current code.